pyaerocom/aeroval/modelmaps_engine.py

```diff
@@ -137,11 +143,18 @@ def _process_map_var(self, model_name, var, reanalyse_existing):
                 logger.info(f"Skipping processing of {outname}: data already exists.")
                 return []
 
-        freq = self.cfg.time_cfg.main_freq
+        #mainfreq = self.cfg.time_cfg.main_freq
+        allfreqs = self.cfg.time_cfg.freqs
         tst = TsType(data.ts_type)
-        if tst < freq:
+        allfreqs = [TsType(fq) for fq in freqs]  # convert them into TsType objects
+        allfreqs.sort(
+            reverse=True
+        )  # sort them from least coarse to coarsest, e.g. [hourly, daily, monthly]
+        coarsestfreq = allfreqs[-1]
+
+        if tst < coarsestfreq:
             raise TemporalResolutionError(f"need {freq} or higher, got{tst}")
-        elif tst > freq:
+        elif tst > coarsestfreq:
```

Comment:
what happens when `tst == coarsestfreq`?

Reply:
also what is the difference between `coarsestfreq` and `freq`?
Don't you want to resample as follows?
``` python
            data = data.resample_time(coarsestfreq)
```

Reply:
there's no need for resampling I guess

Reply:
I did want to have `data = data.resample_time(coarsestfreq)` in the previous version, typo 
with your edits, that code is not needed anymore